deploy/kubernetes/rbac.yaml

```diff
@@ -53,6 +53,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csistoragecapacities"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
"patch" is currently not needed, but I'd rather have it listed to avoid a major new release when we have to add it later.